tests/internal/test_settings.py

```diff
@@ -117,15 +108,15 @@ def _deleted_rc_config():
         },
     ],
 )
-def test_settings(testcase, config, monkeypatch):
+def test_settings_asdf(testcase, config, monkeypatch):
```

Comment:
What does this name mean?

Reply:
lol I had renamed it so that the `-k` option to pytest would pick up only this test and not other tests that matched `test_settings`